MerlinAU.sh

```diff
@@ -2790,11 +2798,24 @@ Please manually update to version $minimum_supported_version or higher to use th
     Say "Required RAM: ${required_space_kb} KB - RAM Free: ${freeRAM_kb} KB - RAM Available: ${availableRAM_kb} KB"
     check_memory_and_prompt_reboot "$required_space_kb" "$availableRAM_kb"
 
+    ##----------------------------------------##
```

Comment:
This is very cool, I like it, you get to compare the contents with what it extracts in the logs :)